### PR TITLE
Upgraded dependencies and replaced deprecated pedantic package with lints

### PR DESCRIPTION
**Pull request by FredrikBorgstrom**
Dependencies were updated. Tested and works fine :)

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
+  collection: ^1.16.0
   console: ^4.1.0
-  test: ^1.17.3
-  collection: ^1.15.0
-  yaml: ^3.1.0
-  petitparser: ^4.0.2
   html_unescape: ^2.0.0
-
+  http: ^0.13.5
+  lints: ^2.0.1
```
Comment (justkawal) [inline pubspec.yaml:16]: Can we also move lints to dev_dependencies ?

File: pubspec.yaml [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
 
 environment:
-  sdk: ">=2.12.0-0 <3.0.0"
+  sdk: ">=2.16.1 <3.0.0"
 
 dependencies:
-  path: ^1.8.0
-  args: ^2.1.0
-  http: ^0.13.2
+  args: ^2.3.1
```
Comment (justkawal) [inline pubspec.yaml:11]: Nice, you sorted the dependencies 🔥.

**Review by justkawal — APPROVED** [review]
LGTM, Thanks

Sorry for late review.

**Comment by FredrikBorgstrom** [comment]
Yes, absolutely. I missed that, lints should be moved under
dev_dependencies.
Thanks for adding my update and thanks for this great utility, it saved me
a lot of time :)

**Pull request merged**